common/cstorpoolcluster/builder.go

```diff
@@ -227,6 +223,12 @@ func (b *Builder) buildDesiredRAIDGroupsByHostName(nodeName string) []interface{
 		var raidGroupList []interface{}
 		var raidGroup []string
 
+		// Stripe pool contains only one raid group.
```

Comment:
can we rephrase this comment as follows:
`Stripe pool implies only one raid group that contains all the disks`

Reply:
@shovanmaity Do we have relevant UT that verify Stripe to only result into a single raid group versus the earlier concept of multiple raid groups?

Reply:
We have unit tests for the function which calls it. There we have test cases for different raid types with different no of block devices.

Reply:
Though test cases are covering by the caller function we can add unit tests for this function also. Please confirm I will add then.

Reply:
Please add them. Make them readable. Spend time on thinking how to add several corner testcases.